### PR TITLE
doc: fix external API documentation

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

**Comment by saghul** [comment]
Jenkins, test this please.

**Comment by saghul** [comment]
Running jenkins again, but all failures must be unrelated since this PR only touches docs.

**Comment by gpolitis** [comment]
unrelated indeed, it seems ICE is failing to establish

File: doc/api.md [hunk 1]
```diff
@@ -21,195 +21,203 @@ The next step for embedding Jitsi Meet is to create the Jitsi Meet API object
...
-
-```
+The `commands` parameter is an object with the names of the commands as keys and the arguments for the commands asvalues:
+```javascript
 api.executeCommands({displayName: ['nickname'], toggleAudio: []});
 ```
 
-You can add event listeners to the embedded Jitsi Meet using ```addEventListener``` method.
-```
+You can add event listeners to the embedded Jitsi Meet using the `ddEventListener` method.
```
Comment (hristoterezov) [inline doc/api.md]: I think there is a typo here - s/ddEventListener/addEventListener

**Pull request merged**